src/dispatch/plugins/dispatch_jira/plugin.py

```diff
@@ -191,58 +192,40 @@ def link_issues(client: Any, link_type: str, issue_id_a: str, issue_id_b: str):
     }
 
 
-def get_incident_data(client: Any, filter: str, start_marker: str, end_marker: str):
-    incident_data = []
-    issues_in_project = client.search_issues(filter)
-    for issue in issues_in_project:
-        description = ""
-        if issue.fields.description:
-            start_index = issue.fields.description.find(start_marker) + len(start_marker)
-            end_index = issue.fields.description.find(end_marker)
-            description = issue.fields.description[start_index:end_index].strip()
-
-        incident_data.append(
-            {"key": issue.key, "title": issue.fields.summary, "summary": description}
-        )
-    return incident_data
-
-
-def get_issue(client: Any, key: str):
-    return client.issue(key)
-
-
 def get_user_name(email):
-    """Attempts to get username."""
+    """Returns username part of email, if valid email is provided."""
     if "@" in email:
         return email.split("@")[0]
     return email
```

Comment:
Is there any more `email` validation than this, elsewhere?

Reply:
Not that I'm aware of. We could get fancier, but not sure it's worth it.